delfin/drivers/ibm/ds8k/ds8k.py

```diff
@@ -259,3 +263,82 @@ def clear_alert(self, context, alert):
     @staticmethod
     def get_access_url():
         return 'https://{ip}:{port}'
+
+    def list_storage_hosts(self, context):
+        try:
+            host_list = []
+            hosts = self.rest_handler.get_rest_info('/api/v1/hosts')
```

Comment:
It is recommended that the URL path be maintained in constants.

Reply:
done 
![image](https://user-images.githubusercontent.com/79631193/155645696-71d91c85-40b7-4ba9-b34c-e372724806a0.png)
![image](https://user-images.githubusercontent.com/79631193/155645720-b94414e1-fd6f-4a06-8f69-a26e28395006.png)